decentraland/unity-explorer
# feat: non compatible wearables in backpack grid

## What does this PR change?

Fixes #507 

Adds the visualization of wearables in the backpack grid that are not compatible with the current body shape.

## How to test the changes?

1. Launch the explorer
2. Open the backpack
3. Go through different wearable categories
4. Check that those wearables which are not compatible with your body shape, should display a special UI state
5. Try changing your body shape
6. The reported issue #507 should not happen anymore

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
